tests/phpunit/tests/block-supports/layout.php

```diff
@@ -251,6 +252,27 @@ public function data_layout_support_flag_renders_classnames_on_wrapper() {
 				),
 				'expected_output' => '<div class="wp-block-group"><div class="wp-block-group__inner-wrapper is-layout-flow wp-block-group-is-layout-flow"></div></div>',
 			),
+			'block with child layout'                      => array(
+				'args'            => array(
+					'block_content' => '<p>Some text.</p>',
+					'block'         => array(
+						'blockName'    => 'core/paragraph',
+						'attrs'        => array(
+							'style' => array(
+								'layout' => array(
+									'columnSpan' => '2',
+								),
+							),
+						),
+						'innerBlocks'  => array(),
+						'innerHTML'    => '<p>Some text.</p>',
+						'innerContent' => array(
+							'<p>Some text.</p>',
+						),
+					),
+				),
+				'expected_output' => '<p class="wp-container-content-1">Some text.</p>', // The generated classname number assumes `wp_unique_id` will not have run previously in this test.
```

Comment:
Tiny nit: the code now uses `wp_unique_prefixed_id` so should the comment refer to that now, too?

```suggestion
				'expected_output' => '<p class="wp-container-content-1">Some text.</p>', // The generated classname number assumes `wp_unique_prefixed_id` will not have run previously in this test.
```

Reply:
It's probably worth including the prefix in Andrew's comment too, `wp_unique_prefixed_id( 'wp-container-content-' )`, as the counter is unique to each prefix.

Reply:
Done, thanks!